docs/create-packages/Package-authoring-best-practices.md

```diff
@@ -94,45 +95,22 @@ See the [.NET library versioning guide](/dotnet/standard/library-guidance/versio
 
 ✔️ DO use the author field for your or your organization's "pretty name."
 > For example, if my NuGet.org username is "jdoe" then using "Jane Doe" for the author field may help consumers recognize me as an author. If my organization's NuGet.org username is "ContosoToolkit" then using "Contoso Corporation" may be more recognizable and inspire more consumer trust.
+
 ### Description
 
 ✔️ DO include a short description (up to 4000 characters) to describe your package.
 > Package descriptions are one of the most prominent fields surfaced in NuGet search and will likely be the first thing potential consumers looks at to determine if a package is right for them.
 
 ### Copyright
 
-✔️ CONSIDER copyrighting your package with "Copyright (c) <name/company\> <year\>."
->A copyright notice essentially indicates that your work cannot be copied without your permission. Including a copyright notice in your package is easy and won't do any harm!
+✔️ DO copyrighting your package with "Copyright (c) <name/company\> <year\>."
```

Comment:
"Do copyrighting" sounds weird.

"DO add a copyright notice to your package ...."?

Reply:
Good catch, will fix 👍

Reply:
```suggestion
✔️ DO add a copyright notice to your package with "Copyright (c) <name/company\> <year\>."
```